masterlist.yaml

```
@@ -18091,6 +18108,15 @@ plugins:
     after:
       - 'Qw_ACE_CRF Patch.esp'
       - 'Qw_WACCF_CRF Patch.esp'
+          
+  - name: 'UltimateCollege.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/44797' ]
```

To match sorting rules for `OCW_CellSettings.esp`.
```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/44797' ]
    group: *cellSettingsGroup
    after:
      - 'ELE_SSE.esp'
      - 'ELFXEnhancer.esp'
      - 'ELFX - Hardcore.esp'
      - 'Immersive Music.esp'
      - 'Immersive Music Replacer.esp'
      - 'Luminosity Lighting Overhaul.esp'
      - 'Luminosity - Skyrim is Lit Edition.esp'
      - 'Soulmancer Soundtrack - NoVindDun.esp'
      - 'Unique Region Names.esp'
      # Group afters, to prevent sorting errors
      - 'MissingEZs_AllExteriors.esp'
      - 'MissingEZsFixed.esp'
      - 'NoGrassINCaves.esp'
```